warp/Network/Wai/Handler/Warp/HTTP2/Receiver.hs

```diff
@@ -232,9 +232,12 @@ control FrameGoAway _ _ Context{controlQ} = do
 
 control FrameWindowUpdate header bs Context{connectionWindow} = do
     WindowUpdateFrame n <- guardIt $ decodeWindowUpdateFrame header bs
-    !w <- (n +) <$> atomically (readTVar connectionWindow)
+    !w <- atomically $ do
```

Comment:
Thanks. I've rebased (and made the commit message more precise for the 2nd commit).